src/python/pants/task/simple_codegen_task.py

```diff
@@ -0,0 +1,342 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Yeah, nothing in that file changed. AFAIK the goal of the copyright year is to establish prior authorship, so you want the earliest year than can reasonably be reasonable...